src/main/groovy/org/mockito/release/gradle/ReleaseConfiguration.java

```diff
@@ -256,6 +257,21 @@ public String getTagPrefix() {
         public void setTagPrefix(String tagPrefix) {
             configuration.put("git.tagPrefix", tagPrefix);
         }
+
+
+        /**
+         * Postfix which will be added to the commit message.
```

Comment:
"Postfix which will be added to the commit message." -> "Text which will be included in the commit message for all commits automatically created by the release automation"

This will make it a bit more clearer for the users.